lib/amqp/exchange.ex

```diff
@@ -204,6 +204,18 @@ defmodule AMQP.Exchange do
     declare(channel, exchange, :topic, options)
   end
 
+  @doc """
+  Convenience function to declare an Exchange of type `headers`.
+
+  ## Options
+
+  This function takes the same options as `declare/4`.
+  """
+  @spec headers(Channel.t(), Basic.exchange(), keyword) :: :ok | Basic.error()
+  def headers(%Channel{} = channel, exchange, options \\ []) do
+    declare(channel, exchange, :headers, options)
+  end
```

Comment:
This level of utility should belong to application level. Can you remove this function and simply use `Exchange.declare/4` instead?